mat/symmetric.go

```diff
@@ -176,6 +197,11 @@ func (s *SymDense) reuseAs(n int) {
 	}
 }
 
+func (s *SymDense) reuseAsZeroed(n int) {
```

Comment:
This should be fully spelled out to avoid double work on the backing slice.

Reply:
Done.